minerl/herobraine/wrappers/video_recording_wrapper.py

```
@@ -0,0 +1,55 @@
+import os
+
+import numpy as np
+from gym import Wrapper
+import cv2
+
+
+class VideoRecordingWrapper(Wrapper):
+    def __init__(self, env, outfile=None, outdir=None):
+        """
+
+        """
+        super().__init__(env)
+        assert outfile or outdir, "Must include somewhere to save the video."
+        assert not (outfile and outdir), "Can only choose one way to save videos."
+        self.outfile = outfile
+        self.outdir = outdir
+        # Used for directory saving.
+        self.current_index = 0
+
```

Fail fast by checking whether 'pov' exists in the observation space dict